README.md

```diff
@@ -1,36 +1,36 @@
 
-PY_PG_TOOLS:
+# Py_pg_tools
 
     A powerful tool to use PostgreSQL in an easier and faster way. It allows
     the possibility to:
 
-    - Change the owner of a database and all of its tables.
-    - Make a custom backup of a PostgreSQL database or a whole cluster.
-    - Delete a PostgreSQL database.
-    - Get some information about some PostgreSQL databases, users or its
+    * Change the owner of a database and all of its tables.
```

Comment:
Dashes (-) work well in markdown for bullets. In fact, I find them less dangerous than asterisks, as they are used for italic (single) or bold (double).